docs/user-guide/knative.md

```diff
@@ -0,0 +1,55 @@
+# Knative Integration
+
+[Knative](https://knative.dev/) is is a popular Kubernetes-based platform for managing serverless workloads.
+
+Replacing Istio with Ambassador can potentially reduce the operational overhead of running Knative. Ambassador can watch for changes in Knative configuration in your Kubernetes cluster and set up routing accordingly.
+
+## Getting started
```

Comment:
I think it would be worthwhile to explain Knative at a very high level here. The reason I believe this is because the Ambassador integration is only for one portion of Knative (the `Knative Serving` portion) and we may get people here who do not know much enough about Knative to know this will not do everything they need.

Reply:
Makes sense. Added. LGTY?

Reply:
Looks great!